deepcell_label_utils/utils.py

```diff
@@ -93,14 +99,100 @@ def int_coords(x):
     return img_mask
 
 
+class DCLZipConverter:
+    """Label converter class for DeepCell Label zip to usable formats."""
+
+    def __init__(self, zip_path=None, zip_obj=None):
+        """
+        Load in a file path for a zip exported from DeepCell Label and
+        automatically reshape to BTZYXC.
+        """
+        if zip_path is not None:
+            zf = zipfile.ZipFile(zip_path, 'r')
+        elif zip_obj is not None:
+            zf = zip_obj
+        else:
+            print("No zip specified...")
+            return
```

Comment:
Tightening up the if statements by relying on pythons behavior of treating objects that are not None as True. Also fail explicitly instead of returning if no inputs are present.

Let's also add an error if both variables are defined.

```suggestion
        if zip_path:
            zf = zipfile.ZipFile(zip_path, 'r')
        elif zip_obj:
            zf = zip_obj
        else:
            raise InputError('Either zip_path or zip_obj must be specified')
```